src/SourceGenerators/Uno.UI.SourceGenerators.Tests.Shared/Given_XBindRewriter.cs

```diff
@@ -1,57 +0,0 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Uno.UI.SourceGenerators.XamlGenerator.Utils;
-
-namespace Uno.UI.SourceGenerators.Tests
-{
-	[TestClass]
-	public class Given_XBindRewriter
-	{
-		[TestMethod]
-
-		// DataTemplates (with context)
-		[DataRow("ctx", "MyProperty.A", "ctx.MyProperty.A")]
-		[DataRow("ctx", "MyProperty", "ctx.MyProperty")]
-		[DataRow("ctx", "MyStaticProperty", "ctx.MyStaticProperty")]
-		[DataRow("ctx", "MyStaticMethod()", "ctx.MyStaticMethod()")]
-		[DataRow("ctx", "MyProperty.A.ToLower()", "ctx.MyProperty.A.ToLower()")]
```

Comment:
Not identical scenario, but this should be covering it.

https://github.com/unoplatform/uno/blob/231e604137b160ecb5f45a1a657c7db7175c865b/src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Controls/DataTemplate_Control.xaml#L16-L17